com.trollworks.gcs/src/com/trollworks/gcs/weapon/WeaponDamage.java

```diff
@@ -29,12 +29,15 @@
 import com.trollworks.gcs.utility.json.JsonWriter;
 import com.trollworks.gcs.utility.text.Enums;
 import com.trollworks.gcs.utility.text.Numbers;
+import com.trollworks.gcs.settings.DamageProgression;
 
 import java.io.IOException;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
 
+import javax.print.attribute.standard.Sides;
```

Comment:
It is unclear to me why you're pulling in the Sides option for the print dialog. Please remove this.